app/styles/components/task-list-cards.scss

```diff
@@ -1,35 +1,57 @@
 .task-list-container {
+  box-sizing: border-box;
+  display: inline-block;
+  height: 100%;
+  margin-left: 7px;
+  white-space: nowrap;
+  width: 280px;
+
   &:first-child {
-    .task-list {
-      margin-left: 10px;
-    }
+    margin-left: 10px;
   }
 
   &:last-child {
-    .task-list {
-      margin-right: 10px;
-    }
+    margin-right: 10px;
   }
 }
 
 .task-list {
   border: 1px solid #DFDFDF;
   background-color: #F7F7F7;
   border-radius: 4px;
-  margin-left: 7px;
+  display: flex;
+  flex-direction: column;
   max-height: 100%;
-  overflow-x: hidden;
-  overflow-y: scroll;
-  padding: 8px 6px;
+  position: relative;
```

Comment:
is this `position: relative` needed with no other position declared on the `.task-list`?

Reply:
I believe so, thanks to some of the flex behavior on children.